tests/MultipleInputParamTest.php

```
@@ -0,0 +1,65 @@
+
+    public function testWithDefaults(): void
+    {
+        $params = ['query' => 'smartphone'];
+
+        $args = $this->namedParameter->getParameters([$this->multipleInputTest, 'onGet'], $params);
+        /** @phpstan-ignore-next-line Parameter type mismatch, runtime provides correct types */
+        $ro = $this->multipleInputTest->onGet(...array_values($args));
+        assert(is_array($ro->body));
+
+        $this->assertSame(200, $ro->code);
+        $this->assertSame('smartphone', $ro->body['search']['query']);
```

PHPStan (2.1.15)

Cannot access offset 'query' on mixed.


(offsetAccess.nonOffsetAccessible)